src/org/camunda/latera/kotlin/bss/connectors/JasperReport.kt

```diff
@@ -0,0 +1,10 @@
+package org.camunda.latera.bss.connectors
+
+import org.camunda.latera.bss.logging.SimpleLogger
+
+class JasperReport {
+
+  fun  testFunction() {
```

Comment:
Extra line and spaces after `fun` :)
It's not so important in this case but I hope we will bring linter in ASAP.